tests/sftp/sensors/test_sftp.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from airflow.exceptions import TaskDeferred
+
+from astronomer.providers.sftp.sensors.sftp import SFTPSensorAsync
+from astronomer.providers.sftp.triggers.sftp import SFTPTrigger
+
+
+def context():
```

Comment:
do we need this? what about doing something like https://github.com/astronomer/astronomer-providers/blob/main/tests/core/sensors/test_external_task.py#L20